app/scripts/views/coppa/coppa-age-input.js

```diff
@@ -56,6 +56,25 @@ define(function (require, exports, module) {
       age.val(age.val().substr(0, 3));
     },
 
+    onKeyDown: function (event) {
+      // helper function to check for digit
+      function isKeyADigit (keyCode) {
+        if ((keyCode > 8 && keyCode < 48) || (keyCode > 57 && keyCode < 96) || keyCode > 105) {
```

Comment:
This would read more clearly if the two ranges (normal top-row digits and keypad digits) were checked explicitly and all other keyCodes (keyCode 8 for example) were ignored:

```js
return ((keyCode >= 48 && keyCode <= 57) || // digit row
        (keyCode >= 96 && keyCode <= 105)); // keypad
```